app/_posts/2020-06-04-improving-search.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+author: kelly-fitzpatrick
+date: 2020-06-04
+title: "Searching Case Resources in H2O: An Update "
+---
+
+We’ve made an update that improves how you can search cases in H2O.
+
+In H2O, you have access to 6.7 million cases made available by the Caselaw Access Project. This update lets you search cases by name, not just citation, when adding resources to your casebook. 
+
+We’ve also made some small accessibility updates and a quick preview link to improve your experience. This round of updates will make it easier to add cases to the resources you create.  
+
+H2O is a platform for creating freely available, high quality, online casebooks and more. Do you have ideas or questions about how your students or faculty can use H2O? You can reach us at [info@opencasebook.org](mailto:info@opencasebook.org)
```

Comment:
I'd skip the last paragraph. It's not necessary to explain what H2O is in this blog.

Reply:
(Or maybe include the contact information.)

Reply:
Fair.
```suggestion
```